chain/client/src/client_actor.rs

```diff
@@ -1508,6 +1512,38 @@ impl ClientActor {
             },
         );
     }
+
+    /// Periodically compute RocksDB stats.
+    fn compute_rocksdb_stats(&self, ctx: &mut Context<Self>) {
+        if let Some(period) = self.client.config.compute_rocksdb_stats_period {
+            near_performance_metrics::actix::run_later(ctx, period, move |act, ctx| {
+                info!(target: "neard", "Computing rocksdb stats...");
+                let rocksdb_stats = get_rocksdb_stats(
+                    self.client.chain.store().store().get_rocksdb().unwrap().path(),
+                );
```

Comment:
Hm, this looks like a rather scary thing to do...

My understanding is what happens here is that once in a while we are going to run external process to analyse database, while the database is still in use. 

The biggest immediate thing that'll go wrong is that this processing would completely block client's actor loop -- no other stuff will be happening while the sst dump is in progress. 

More generally, it doesn't seem like `sst_dump` is a thing which is intended to be run in production like this, so I think this can add a fair amount of unknown unknowns. 

My gut feel is that these stats should be computed and send by a different process altogether -- rather than running the thing from within a live node, setup some cron job to run a script periodically (the script *can* be `neard send-db-telemety `~/.neard`. It's OK to keep the code for this in the same binary).

I see however how  this can be operationally more convenient if we just do this from within the process itself... As a litmus test, the way we intend to use this code, would it be ok if it outright panicked? If we are OK with that, than, yeah, probably fine to keep this. Although, even in that case, rather than spawning this from-within a clinet, I'd spawn a whole dedicated thread for this from `main`, like we do here: 

https://github.com/near/nearcore/blob/93efd262fff035d9825898bd9e906fb9efcf19af/neard/src/main.rs#L48

Reply:
+1 to this comment. We should try running those stats from the outside.